thorlcr/activities/msort/thsortu.cpp

```diff
@@ -1971,7 +1971,7 @@ class CMultiCoreUnorderedJoinHelper: public CMultiCoreJoinHelperBase
     virtual void stop()
     {
         CMultiCoreJoinHelperBase::stop();
-        workqueue.stop();
+        workqueue.stop(1, numworkers);
```

Comment:
There is 1 writer (via addWork), and N readers, I think this should be workqueue.stop(numWorkers, 1); ?